vshard/replicaset.lua

```diff
@@ -680,7 +680,7 @@ local function replicaset_update_master(replicaset, old_master_uuid,
     if old_master_uuid == candidate_uuid then
         -- It should not happen ever, but be ready to everything.
         log.warn('Replica %s in replicaset %s reports self as both master '..
-                 'and not master', master_uuid, replicaset_uuid)
+                 'and not master', old_master_uuid, replicaset_uuid)
```

Comment:
I suggest to add a test case for this error

Reply:
I agree with Oleg. Like in the other PR, you would need to add a test. Look at the end of `master_discovery.test.lua`. You also need to make the commit message conform with rules. Here it would be `router: fix undeclared var usage`.